tests/gnmi/helper.py

```diff
@@ -40,19 +54,32 @@ def apply_cert_config(duthost):
     dut_command += "--ca_crt /etc/sonic/telemetry/gnmiCA.pem -gnmi_native_write=true -v=10 >/root/gnmi.log 2>&1 &\""
     duthost.shell(dut_command)
     time.sleep(GNMI_SERVER_START_WAIT_TIME)
+    dut_command = "sudo netstat -nap | grep %d" % env.gnmi_port
+    output = duthost.shell(dut_command, module_ignore_errors=True)
+    if "telemetry" not in output['stdout']:
```

Comment:
>telemetry [](http://example.com/codeflow?start=8&length=9)

If we rename the process name to "gnmi", this kind of issue will reappear. Could you get the main process name programmatically?

Reply:
“netstat -nap” shows the process name, and process name is "telemetry" for both gnmi container and telemetry container.

Reply:
I have created ADO 26532128 to rename process name, and I have fixed this test